### PR TITLE
Aso Bid Adapter: support schain

### PR DESCRIPTION
**Pull request by adserver-online**
## Type of change

- [x] Feature
- [x] Code style update
- [x] Refactoring

## Description of change

1. schain support added
2. previously internal parameter 'serverUrl' was renamed to 'server' and published
3. internal parameter 'serverPath' was removed

## Other information
Docs PR https://github.com/prebid/prebid.github.io/pull/4075

### CODE REVIEW

File: modules/asoBidAdapter.js [hunk 1]
```diff
@@ -19,8 +20,15 @@ const BIDDER_CODE = 'aso';
 const DEFAULT_SERVER_URL = 'https://srv.aso1.net';
 const DEFAULT_SERVER_PATH = '/prebid/bidder';
 const OUTSTREAM_RENDERER_URL = 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
+const VERSION = '$prebid.version$_1.1';
 const TTL = 300;
 
+/*
+ * In case you're Adserver.Online platform's client who needs a branded adapter,
+ * WE KINDLY ASK YOU DO NOT COPY THIS ADAPTER UNDER NEW NAME!
```
Comment (patmmccann) [inline modules/asoBidAdapter.js:28]: This isn't necessary, please remove

File: modules/asoBidAdapter.js [hunk 2]
```diff
@@ -273,10 +281,8 @@ function createVideoImp(bidRequest, videoParams) {
 }
 
 function getEnpoint(bidRequest) {
-  const serverUrl = bidRequest.params.serverUrl || DEFAULT_SERVER_URL;
```
Comment (patmmccann) [inline modules/asoBidAdapter.js:276]: Note typo in endpoint above

File: modules/asoBidAdapter.md [hunk 3]
```diff
@@ -14,12 +14,11 @@ For more information, please visit [Adserver.Online](https://adserver.online).
...
-| Name          | Scope    | Description             | Example   | Type      |
-|---------------|----------|-------------------------|-----------|-----------|
-| `zone`        | required | Zone ID                 | `73815`   | `Integer` |
-| `attr`        | optional | Custom targeting params | `{keywords: ["a", "b"]}` | `Object` |
-
-
+| Name      | Scope    | Description             | Example                  | Type       |
+|-----------|----------|-------------------------|--------------------------|------------|
+| `zone`    | required | Zone ID                 | `73815`                  | `Integer`  |
+| `attr`    | optional | Custom targeting params | `{keywords: ["a", "b"]}` | `Object`   |
```
Comment (patmmccann) [inline modules/asoBidAdapter.md:20]: Please note that if you take keywords from bid parameters, you must also take them from ortb2 object

**Review by patmmccann — CHANGES_REQUESTED** [review]
Thanks for your submission!

**Comment by adserver-online** [comment]
@ChrisHuie could you please kick off tests yet again? they stuck with the same error "Could not find a usable config.yml..."

**Review by patmmccann — APPROVED** [review]

**Pull request merged**